invenio_queues/__init__.py

```diff
@@ -6,7 +6,126 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Invenio module for collecting statistics."""
+
+r"""Invenio module for managing queues.
+
+This guide will show you how to get started with Invenio-Queues. It assumes
+that you already have knowledge of Flask applications and Invenio modules.
+
+It will then explain key topics and concepts of this module.
+
+Getting started
+---------------
+
+You will learn how to register queues and to interact with it. To begin with,
+you need to setup your virtual environment and install this module.
```

Comment:
```suggestion
You will learn how to register queues and to interact with them. To begin with,
you need to setup your virtual environment and install this module.
```

Not sure if it's taken from another module's docs, but here I would also expect commands on how to setup my virtualenv and install the module. Otherwise, this can be skipped or say something in the lines of "we assume you've setup a virtualenv and installed this module".